DrawerKit/DrawerKit/Internal API/PresentationController.swift

```diff
@@ -200,91 +318,63 @@ private extension PresentationController {
         animator.startAnimation()
     }
 
-    func addCornerRadiusAnimationEnding(at endPositionY: CGFloat, clamping: Bool = false) {
-        guard maximumCornerRadius > 0 && drawerPartialY > 0 && endPositionY != currentDrawerY else { return }
-
-        let endPosY = (clamping ? clamped(endPositionY) : endPositionY)
-        guard endPosY != currentDrawerY else { return }
+    func addCornerRadiusAnimationEnding(at endState: DrawerState) {
+        guard maximumCornerRadius != 0 && drawerPartialY != 0
+            && endState != currentDrawerState else { return }
```

Comment:
indentation per two comments above

Reply:
Fixed in https://github.com/Babylonpartners/DrawerKit/pull/27/commits/c83e68939ca6999a0615f90925bf554f75c8bfc9.